imports/plugins/core/accounts/server/no-meteor/mutations/addAccountToGroup.js

```diff
@@ -0,0 +1,117 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import { difference } from "lodash";
+import SimpleSchema from "simpl-schema";
+import ensureRoles from "../util/ensureRoles";
+
+const inputSchema = new SimpleSchema({
+  accountId: String,
+  groupId: String
+});
+
+/**
+ * @name accounts/addAccountToGroup
+ * @memberof Mutations/Accounts
+ * @method
+ * @summary Add an account to a group
+ * @param {Object} context - GraphQL execution context
+ * @param {Object} input - Input arguments
+ * @param {String} input.accountId - The account ID
+ * @param {String} input.groupId - The group ID
+ * @return {Promise<Object>} with updated address
+ */
+export default async function addAccountToGroup(context, input) {
+  inputSchema.validate(input);
+
+  const { accountId, groupId } = input;
+  const {
+    appEvents,
+    collections: {
+      Accounts,
+      Groups,
+      users
+    },
+    user
+  } = context;
+
+  const group = await Groups.findOne({ _id: groupId });
+  if (!group) throw new ReactionError("not-found", "No group found with that ID");
+
+  const { permissions: groupPermissions = [], shopId } = group;
+
+  // An account can add another account to a group as long as the person adding
+  // has all permissions granted by that group.
+  // We can't use `userHasPermission` here because we want to make sure they
+  // have ALL the permissions rather than ANY.
+  if (!context.isInternalCall && difference(groupPermissions, user.roles[shopId] || []).length > 0) {
```

Comment:
Do we need to import`lodash` here for this?

Could we do something like `groupPermissions.filter((permission) => !user.roles[shopId].includes(permission))` instead?

I know at one point our policy was "lodash only if needed", if that's changed, then `difference` is cleaner looking.

Reply:
No, lodash isn't needed.